src/SASViyaApiClient.ts

```diff
@@ -381,7 +387,17 @@ export class SASViyaApiClient {
         )
         const newFolderName = `${parentFolderPath.split('/').pop()}`
         if (newParentFolderPath === '') {
-          throw new Error('Root folder has to be present on the server.')
+          let error: string = `Root folder ${parentFolderPath} was not found\nPlease check ${this.serverUrl}/SASDrive\nIf folder DOES exist then it is likely a permission problem\n`
+          if (accessToken) {
+            const decodedToken: DecodedToken = decodeToken(accessToken)
+            const scope = decodedToken.scope
+            error =
+              error + `The following scopes are contained in access token:\n`
+            scope.forEach((element: any) => {
```

Comment:
`error += scope.join('\n')` would be better in my opinion.

Reply:
then we wouldn't be able to add star or dash as prefix. If prefix is necessary we would have to add another statement as
`scope = scope.map(element => '* ' + element)`
what do you suggest?

Reply:
Ah, apologies, I didn't notice the `*`.
Yes, I think the `map` and `join` approach is preferable since we will only be adding to the `error` string once instead of multiple times.